using/config.json

```diff
@@ -6,5 +6,12 @@
     "path": "using/README.md",
     "title": "Using Exercism",
     "blurb": "Learn how to make the most of our experience on Exercism"
+  },
+  {
+    "uuid": "7b753e2a-bd68-4d68-bdd8-c356de4342b6",
+    "slug": "solving-exercises/locally",
+    "path": "using/solving-exercises/locally.md",
+    "title": "Solving Exercises Locally",
+    "blurb": "Learn how to solve exercises on your local machine"
```

Comment:
I haven't yet added entries for the other two files that this PR adds, as I only want to add them once the "Solving Exercises Online" document has been written.